src/ck-core/charm++.h

```diff
@@ -70,62 +70,41 @@ class CkEntryOptions : public CkNoncopyable {
 	int queueingtype; //CK_QUEUEING type
 	int prioBits; //Number of bits of priority to use
 	typedef unsigned int prio_t; //Datatype used to represent priorities
-	prio_t *prioPtr; //Points to message priority values
+	std::vector<prio_t> prioPtr; //Points to message priority values
```

Comment:
Thinking about this more, the change to std::vector here will bloat the size of CkEntryOptions some. So, worth thinking about this more. Maybe we can get rid of prioBits and prioStore now, instead using the vector? Or maybe we should leave it as is?

Reply:
If `std::vector` has drawbacks here then maybe we should leave it like it is. One big focus of #1314 is eliminating `if (p.isUnpacking) myArray = new ...` from PUP routines throughout the runtime and that is what this PR does.

Reply:
I agree